Utilities/bootstrap

```diff
@@ -174,6 +174,7 @@ def parse_global_args(args):
     args.swift_argument_parser_source_dir = os.path.join(args.project_root, "..", "swift-argument-parser")
     args.swift_driver_source_dir = os.path.join(args.project_root, "..", "swift-driver")
     args.swift_crypto_source_dir = os.path.join(args.project_root, "..", "swift-crypto")
+    args.swift_collections_source_dir = os.path.join(args.project_root, "..", "swift-collections")
```

Comment:
Same with #3590, changes in this file might conflict with #3582

Reply:
yes, lets get #3582 in first and refactor this on top. we need changes made to swift-collections cmake setup anyways